runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -14665,3 +14665,18 @@ J9::Z::TreeEvaluator::inlineIntegerStringSize(TR::Node* node, TR::CodeGenerator*
 
    return node->setRegister(lengthReg);
    }
+
+TR::Register*
+J9::Z::TreeEvaluator::inlineOnSpinWait(TR::Node *node, TR::CodeGenerator *cg)
```

Comment:
Please add reference / Comment to the JNI call to OnSpinWait that emits NOP for reference to explain why we are generating NOP.